src/Apps/Auction/Components/__tests__/AuctionArtworkFilter.jest.tsx

```diff
@@ -33,16 +33,13 @@ describe("AuctionArtworkFilter", () => {
       return <AuctionArtworkFilterRefetchContainer {...props} />
     },
     query: graphql`
-      query AuctionArtworkFilterTestQuery($input: FilterArtworksInput!) {
```

Comment:
I don't really understand the test failure. I didn't modify anything with this component, just _added_ the query renderer. This test is on the refetch container, not modified in the file above. So why is it failing now?